### PR TITLE
Add remove indicator at pos and line.

### PR DESCRIPTION
**Pull request by jcs090218**
What do you think about this? I think is kind of useful. :/

**Edit:** I don't think will work, what do you think implement this kind of functions?

### CODE REVIEW

**Comment by Fuco1** [comment]
I think this might work.  Did you test it?  I have no problem with merging this functionality.  The code looks OK to me but I don't remember if the indicators are stored at the beginning or end of lines.

You might need to change `pos pos` to `pos (1+ pos)` to make the interval non-empty.

File: indicators.el [hunk 1]
```diff
@@ -367,6 +367,31 @@ more indicators are on the same physical line."
...
+      (dolist (ind ind-managed-absolute-indicators)
+        (let* ((pos (car ind))
+               (mkr-pos (marker-position pos)))
+          (when (and (>= mkr-pos start-pt)
+                     (<= mkr-pos end-pt))
+            (push ind remove-inds))))
+      (dolist (ind remove-inds)
+        (setq ind-managed-absolute-indicators (remove ind ind-managed-absolute-indicators)))
+      (remove-overlays start-pt end-pt 'ind-indicator-absolute t))))
+
```
Comment (jcs090218) [inline indicators.el]: Seems like I have to remove it from `ind ind-managed-absolute-indicators`.

**Review by jcs090218 — COMMENTED** [review]
I think currently this will work with the absolute indicator list, which as `ind ind-managed-absolute-indicators`. Should I make into `optional`?

File: indicators.el [hunk 2]
```diff
@@ -368,15 +368,29 @@ more indicators are on the same physical line."
...
     (goto-char (point-min))
     (forward-line (1- line))
     (ind-remove-indicator (point))))
 
 (cl-defun ind-remove-indicator (pos)
   "Remove the indicator to position POS."
-  (remove-overlays pos pos 'ind-indicator-absolute t))
+  (save-excursion
+    (goto-char pos)
+    (delete-dups ind-managed-absolute-indicators)
```
Comment (Fuco1) [inline indicators.el]: Where is this function coming from?

Comment (jcs090218) [reply indicators.el]: This is from `subr.el` file.

Comment (Fuco1) [reply indicators.el]: Ah okay, I should've checked.  `subr.el` is included automatically so you don't need to require.

**Comment by jcs090218** [comment]
What do you think about this patch?

**Pull request closed without merging**